### ISSUE TITLE
Feature request: "Journal entry" button on the toolbar

### ISSUE THREAD

**Issue by eloemosynator** [gsantner/markor#1117]
#### General information

* **App version: 2.4.0 (119)** 
* **System: Android 11 on Google Pixel 3a** 

#### Description

First of all - thank you for this amazing editor, I'm using it daily! 👍 

Here's a description of my workflow to make things easier to understand:

On my home screen I have a shortcut to a markdown file which I use as a journal of sorts (making timestamped entries). At the moment here's what I need to do to add an entry:

1. Click the shortcut to open the file in Markor
2. Once the file is opened, long-tap the header to go to the top
3. Then long-tap again to go to the bottom (BTW, "Start on bottom" option in settings doesn't work for some reason, the editor just remembers the cursor position)
4. Click the clock icon on the toolbar
5. Click Ok in the date format dialog
6. Hit Enter to add a newline so I could start entering text

So I thought it would be awesome to have a button on the toolbar which would:

1. Go the bottom of the file
2. Insert current timestamp in the last used format followed by a newline

This would greatly simplify my workflow as it would now take only two steps (instead of six) before starting writing:

1. Click the shortcut to open the file in Markor
2. Click the "Journal entry" button

Does it make sense, what do you think?

**Comment by gsantner** [comment]
> Insert current timestamp in the last used format followed by a newline

I use exactly that for kind of journal. PRO tipp: Date button, in the format list the bottom-most template ;).

**Comment by eloemosynator** [comment]
Thank you for the tip, I've now created my own template that adds a newline.

However, that only solves step #6 in my original description. How about eliminating steps 3-5? I mean, it would be nice to have it done just by clicking on a button on the toolbar, and that's what I'm proposing: add a new "Journal entry" button, that would both automatically put the cursor to the bottom and insert the current timestamp without having to display any dialogs?

**Comment by gsantner** [comment]
long press date button = insert with last used format.

I think that together is a good enough solution to what you seek for.
